db/schema.rb

```diff
@@ -10,6 +10,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 2018_09_04_183131) do
+
+  create_table "users", force: :cascade do |t|
+    t.string "sunet_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["sunet_id"], name: "index_users_on_sunet_id"
```

Comment:
Notice here that no unique key has been defined. I wonder if this because you ran `db:migrate` against a sqlplus instead of a postgres.

Reply:
but yeah, maybe it is DB specific?  it's possible that at the time we were still using sqlite and not PG for laptop development (we switched over the PG on laptops too after not that long, but it wasn't a thing we did for local dev from the get go).